src/app/commandlineparser.cpp

```diff
@@ -108,6 +109,7 @@ void CommandLineParser::init()
     m_parser.addOption(QCommandLineOption("fps", "Frame per second [60, 30, 24]", "24"));
     m_parser.addOption(QCommandLineOption("ls", "Pause before playback in seconds (3.0)", "3.0"));
     m_parser.addOption(QCommandLineOption("ts", "Pause before end of video in seconds (3.0)", "3.0"));
+#endif
```

Comment:
I believe that it makes no sense to remove all the code (for what? to speed up the compilation of these few lines?) I agree that it is worth removing only the addition of command options

Reply:
OK, I'm fine with that too.